File: JSBASIC/80_triangle_type.js
// 80	80_triangle_type.js	Determine the type of triangle: equilateral, isosceles, or scalene.

let a = 5, b = 5, c = 8;
console.log("Sides:", a, b, c);
if (a === b && b === c) {
    console.log("Equilateral Triangle.");
} else if (a === b || b === c || a === c) {
    console.log("Isosceles Triangle.");
} else {
    console.log("Scalene Triangle.");
}

File: JavaBasic/80_triangle_type.java
// 80	80_triangle_type.java	Determine the type of triangle: equilateral, isosceles, or scalene.
public class _80_triangle_type {
    public static void main(String[] args) {
        int a = 5, b = 5, c = 8;
        System.out.println("Sides: " + a + ", " + b + ", " + c);
        if (a==b && b==c) System.out.println("Equilateral Triangle.");
        else if (a==b || b==c || a==c) System.out.println("Isosceles Triangle.");
        else System.out.println("Scalene Triangle.");
    }
}
